config/base_boxes.yaml

```diff
@@ -50,13 +46,7 @@ boxes:
         group: 'freeipa_server'
         server: 'centos7-katello-nightly'
 
-  centos7-bats:
-    box: centos7
-    ansible:
-      playbook: 'playbooks/bats_pipeline_nightly.yml'
-      group: 'bats'
-
```

Comment:
is this removal intentional? I'm not sure how it is related to the shells removal

Reply:
It can be removed but you're right that it doesn't belong in this PR.